tests/cupyx_tests/scipy_tests/special_tests/test_convex_analysis.py

```diff
@@ -0,0 +1,58 @@
+import itertools
+import unittest
+
+import cupy
+import numpy
+from cupy import testing
+import scipy.special  # NOQA
+
+import cupyx.scipy.special
+
+
+@testing.gpu
+@testing.with_requires('scipy')
+class TestSpecialConvex(unittest.TestCase):
+
+    def test_huber_basic(self):
+        assert cupyx.scipy.special.huber(-1, 1.5) == cupy.inf
+        testing.assert_allclose(cupyx.scipy.special.huber(2, 1.5),
+                                0.5 * 1.5**2)
+        testing.assert_allclose(cupyx.scipy.special.huber(2, 2.5),
+                                2 * (2.5 - 0.5 * 2))
+
+    @testing.for_dtypes(['e', 'f', 'd'])
```

Comment:
Please fix other places as well.
```suggestion
    @testing.for_float_dtypes()
```

Reply:
Sure. I think this `for_dtypes(['e', 'f', 'd'])` styel was copied from some other tests for `cupyx.scipy.special` if you want to edit there as well in a separate PR